mlir/test/Dialect/MIOpen/ops_2_f16.mlir

```diff
@@ -0,0 +1,232 @@
+// RUN: mlir-opt %s | FileCheck %s
+// RUN: mlir-opt %s | mlir-opt | FileCheck %s
+// Run: mlir-opt -mlir-print-op-generic %s | mlir-opt | FileCheck %s
+
```

Comment:
Can we move those tests to two independent test case each with more descriptive name? One for blockwise ops; Another for threadwise ops.